python/rapidsmpf/CMakeLists.txt

```diff
@@ -10,16 +10,33 @@ include(rapids-cpm)
 rapids_cpm_init()
 include(rapids-cuda)
 include(rapids-find)
+
 rapids_cuda_init_architectures(rapidsmpf-python)
 
+option(BUILD_IN_CONDA "Build RapidsMPF library using conda" ON)
+
 project(
   rapidsmpf-python
   VERSION "${RAPIDS_VERSION}"
   LANGUAGES CXX CUDA
 )
 
-find_package(rapidsmpf REQUIRED "${RAPIDS_VERSION}")
-find_package(ucxx-python REQUIRED "${UCXX_VERSION}")
+if(NOT BUILD_IN_CONDA)
+  set(RELOCATE_LIBRAPIDSMPF ON)
+  set(CUDA_STATIC_RUNTIME ON)
+  add_subdirectory(../../cpp librapidsmpf)
+
+  # TODO: don't use env-vars for this
+  include_directories($ENV{SITE_PACKAGES}/ucxx/include)
```

Comment:
Without this (or similar) we fail at `rapids_find_package(ucxx)` because of complains about not being able to find `ucxx-config.cmake` or similar (but that file IS present, just not where it's looking)